app.yaml

```diff
@@ -4,23 +4,25 @@ threadsafe: true
 
 handlers:
 - url: /ajax/.*
-  script: main.app
+  script: main.APP
```

Comment:
See comment in other PR -- I looked at the app engine docs too and I think it's better to just follow the lowercase convention. OK to add a line exception in the place where `app` is used, if needed.

Reply:
Changes reverted!